jekyll/_cci2/about-circleci.md

```diff
@@ -11,74 +10,83 @@ version:
 
-CircleCI then sends an email notification of success or failure after the tests complete. CircleCI also includes integrated [Slack and IRC notifications]( {{ site.baseurl }}/2.0/notifications). Code test coverage results are available from the details page for any project for which a reporting library is added.
+CircleCI then sends an email notification of success or failure after the tests complete. CircleCI also includes integrated [Slack and IRC notifications]({{site.baseurl}}/2.0/notifications). Code test coverage results are available from the details page for any project for which a reporting library is added.
 
-CircleCI may be configured to deploy code to various environments, including AWS CodeDeploy, AWS EC2 Container Service (ECS), AWS S3, Google Kubernetes Engine (GKE), Microsoft Azure, and Heroku. Other cloud service deployments are easily scripted using SSH or by installing the API client of the service with your job configuration.
+CircleCI may be configured to deploy code to various environments, including:
+- AWS CodeDeploy
+- AWS EC2 Container Service (ECS)
+- AWS S3, Google Kubernetes Engine (GKE)
+- Microsoft Azure
+- Heroku
 
-## What is continuous integration?
-{: #what-is-continuous-integration }
+Other cloud service deployments are easily scripted using SSH or by installing the API client of the service with your job configuration.
 
-**Continuous integration** is a practice that encourages developers to integrate their code into a `main` branch of a shared repository early and often. Instead of building out features in isolation and integrating them at the end of a development cycle, code is integrated with the shared repository by each developer multiple times throughout the day.
+## Benefits of CircleCI
+{: #benefits-of-circleci }
 
-**Continuous Integration** is a key step to digital transformation.
+CircleCI runs nearly one million jobs per day in support of 30,000 organizations. Organizations choose CircleCI because jobs run fast and builds can be optimized for speed.
 
-**What?**  
-Every developer commits daily to a shared mainline.  
-Every commit triggers an automated build and test.  
-If build and test fails, it’s repaired quickly - within minutes.
+CircleCI can be configured to run very complex pipelines efficiently with sophisticated [caching,]({{site.baseurl}}/2.0/caching/) [docker layer caching,]({{site.baseurl}}/2.0/docker-layer-caching/) and [resource classes]({{site.baseurl}}/2.0/optimizations/#resource-class) for running on faster machines.
 
-**Why?**  
-Improve team productivity, efficiency, happiness.  
-Find problems and solve them, quickly.  
-Release higher quality, more stable products.
+As a developer using CircleCI you can:
```

Comment:
I think full stops at the end of these bullets.

Reply:
@rosieyohannan I've not been putting full stops on bullet lists after noticing how often we do/don't use them elsewhere on the site. Definitely maybe something maybe for the style guide. If I add them here, I will also add them to all the bullet points that have a whole or partial sentence structure.

Reply:
Yes! I know we are inconsistent on this (:-() but in my style guide notes I have: use a full stop if the bullet point completes a sentence with the introductory stem. So in this case I think bullet points work.